pkg/algorithm/cell.go

```diff
@@ -24,11 +24,12 @@ package algorithm
 
 import (
 	"fmt"
+
 	"github.com/microsoft/hivedscheduler/pkg/api"
 	"k8s.io/klog"
 )
 
-// A Cell represents a set of GPUs affinitized by their interconnection topology.
+// A Cell represents a set of leaf cells affinitized by their interconnection topology.
```

Comment:
Can we use cell to define cell?
Maybe device is more appropriate here

Reply:
recursive definition? using leaf cell to define cell is like using sub tree to define tree